hovercontrol/src/main.rs

```diff
@@ -13,17 +13,32 @@ use std::time::Duration;
 const BAUD_RATE: u32 = 115_200;
 const SLEEP_DURATION: Duration = Duration::from_millis(2);
 
+// 50 or 75 lets you turn the bar closer to 90 degrees.
 const DEFAULT_SCALE: f32 = 30.0;
 const MAX_SCALE: f32 = 100.0;
 
+// I'm not really sure how to reason about this. At the end, I think you had
+// Right: 'max speed -300..30'
+// Left: 'max speed -30..300'
+// which is a bit confusing, because you have to do the reverse-right dance in
+// your head.
+// Could each half of the board be told which way is "forwards", and then log
+// everything with forwards = positive?
+// Do we ever allow the user to change the max reverse speed?
 const DEFAULT_MAX_SPEED: RangeInclusive<i16> = -200..=30;
```

Comment:
I also noticed that 30 is *just* not enough to turn the motors. Maybe I will tweak this so that it will be able to push rope a bit, but *very* slowly or something.